programs/intent-transfer/src/bridge/processor/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod bridge_ntt_tokens;
```

Comment:
why do we have a one file subfolder? why not just put `bridge_ntt_tokens.rs` into `processor.rs`?